ci-operator/step-registry/edge-nodes/deploy-app/edge-nodes-deploy-app-commands.sh

```diff
@@ -32,6 +36,26 @@ apiVersion: v1
 metadata:
   name: edge-app
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: app
+  namespace: edge-app
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I need to check if we really need this on CI, iirc I see it is setting somewhere